mlos_bench/mlos_bench/config/environments/apps/fio/scripts/remote/latency-profile.fio

```diff
@@ -0,0 +1,21 @@
+# Test job that demonstrates how to use the latency target
+# profiling. Fio will find the queue depth between 1..128
+# that fits within the latency constraints of this 4k random
+# read workload.
+
+[global]
+bs=4k
+rw=randrw
+random_generator=lfsr
+direct=1
+ioengine=libaio
+iodepth=128
+# Set max acceptable latency to 500msec
+latency_target=500000
```

Comment:
That seems really high.  Maybe something lower like 50ms?

Reply:
yep. also, I read the fio docs and this is probably not the best benchmark for our needs. I'll use a config that Microsoft recommends to benchmark our disks in the cloud, from here: [Azure/VMs/Benchmark a disk](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-benchmarks#maximum-read-and-write-iops)